### DISCUSSION TITLE
DM2024-Lab1-Master.ipynb output different from original

### DISCUSSION THREAD

**Discussion by ktpss97094** [didiersalazar/DM2024-Lab1-Master#3] [Q&A] [answered]
Hello:
I run all the inside cells without modifying DM2024-Lab1-Master.ipynb at all. But in cell 288, the filter words I output are almost different from the original output provided. Is this normal? Or does it mean that my environment settings are wrong? thank you

original output:
![image](https://github.com/user-attachments/assets/d3d40694-8209-46f0-a74d-b7fb85635eea)

my output:
![image](https://github.com/user-attachments/assets/cc8febb1-1d61-43d4-8e37-8fe658282848)

**Comment by didiersalazar** [comment, marked as answer]
> Hello:
> I run all the inside cells without modifying DM2024-Lab1-Master.ipynb at all. But in cell 288, the filter words I output are almost different from the original output provided. Is this normal? Or does it mean that my environment settings are wrong? thank you
> 
> original output:
> ![image](https://github.com/user-attachments/assets/d3d40694-8209-46f0-a74d-b7fb85635eea)
> 
> my output:
> ![image](https://github.com/user-attachments/assets/cc8febb1-1d61-43d4-8e37-8fe658282848)
> 

Hi, it is ok, there are more number of words with frequency of 1 so the algorithm might choose different words than mine to filter. But they also represent the same group of least frequent words.

**Reply by ktpss97094** [reply]
ok, thank you!
